src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -401,21 +401,119 @@ class Organization: ADOSVTBase
             
             if(($responseObj | Measure-Object).Count -gt 0 ) #includes both custom installed and built in extensions.
             {
-                $extensionList = $responseObj | Select-Object extensionName,publisherId,publisherName,version,flags # 'flags' is not available in every extension. It is visible only for built in extensions. Hence this appends 'flags' to trimmed objects.
+                $extensionList = $responseObj | Select-Object extensionName,publisherId,publisherName,version,flags,lastPublished,scopes,extensionId # 'flags' is not available in every extension. It is visible only for built in extensions. Hence this appends 'flags' to trimmed objects.
                 $extensionList = $extensionList | Where-Object {$_.flags -notlike "*builtin*" } # to filter out extensions that are built in and are not visible on portal.
+                
+
                 $extCount = ($extensionList | Measure-Object ).Count;
 
                 if($extCount -gt 0)
-                {               
-                    $controlResult.AddMessage("No. of installed extensions: " + $extCount);
+                {   
+                    if([AzSKRoot]::IsDetailedScanRequired -eq $true)
+                    {
+                        # find inactive extensions
+                        $date = Get-Date
+                        $ExtensionsLastUpdatedInYears=$this.ControlSettings.Organization.ExtensionsLastUpdatedInYears;
+                        $thresholddate = $date.AddYears(-$ExtensionsLastUpdatedInYears)
+                        $staleExtensionList = $extensionList | Where-Object {([datetime] $_.lastPublished) -lt $thresholddate}
+                        $controlResult.AddMessage("`nNo. of extensions not have been updated since past $ExtensionsLastUpdatedInYears years : "+ $staleExtensionList.count)                        
```

Comment:
No. of extensions that haven't been published in last $variable years:

Reply:
updated message